.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>hibernate-ogm</name>
```

Comment:
You should not commit this files.
This should also be in the .gitignore so I'm not sure how you did it :)